.env.example

```diff
@@ -0,0 +1,6 @@
+TZ=Pacific/Auckland
```

Comment:
If the API is running locally outside of a container it will pick up the timezone.
If you need the agent also to run in the same timezone you would need to set this and pass in the timezone db.